blivet/formats/disklabel.py

```diff
@@ -324,6 +324,15 @@ def logicalPartitions(self):
         return logicals
 
     @property
+    def primaryPartitions(self):
+        try:
+            primaries = self.partedDisk.getPrimaryPartitions()
```

Comment:
I think this could be simplified to ``return self.partedDisk.getPrimaryPartitions()`` with the ``primaries`` variable being dropped.